### PR TITLE
Help people understand what this bot is about

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: run-bot.py [hunk 1]
```diff
@@ -87,6 +87,8 @@ def handle(msg):
     if msg.get("text", "").startswith("/"):
         welcome = (
             "Hello!\n"
+            "Any message, picture, sound, video sent to me will be displayed "
+            "live on http://leplatrem.github.io/kinto-telegram-wall/\n
```
Comment (leplatrem) [inline run-bot.py]: This should definitely be a config value :)
this bot can be used by anyone with any webpage

Comment (magopian) [inline run-bot.py]: Indeed, will modify the PR asap

**Comment by magopian** [comment]
Added a `WALL_URL` env variable, what do you think?

**Comment by leplatrem** [comment]
Thanks !!

**Pull request merged**